checker/checker.go

```diff
@@ -251,7 +251,10 @@ func (v *visitor) BinaryNode(node *ast.BinaryNode) reflect.Type {
 		if isInteger(l) && isInteger(r) {
 			return reflect.SliceOf(integerType)
 		}
-
+	case "&":
```

Comment:
Defining operators' symbols in const macros would make possible changes easier.

Reply:
Good idea! Thanks